strings/src/main/res/values/untranslated.xml

```diff
@@ -7,7 +7,7 @@ OR CONDITIONS OF ANY KIND, either express or implied. See the License for
 the specific language governing permissions and limitations under the License. -->
 <resources>
     <!-- strings that should not be translated -->
-    <string name="app_name" translatable="false">ODK Collect</string>
```

Comment:
Another library also had an `app_name` and for some reason this was now overriding ours. This should prevent that happening in the future.